packages/agents-hosting-extensions-teams/src/teamsAttachmentDownloader.ts

```diff
@@ -3,90 +3,11 @@
-     */
-  public async downloadFiles (context: TurnContext): Promise<InputFile[]> {
-    // Filter out HTML attachments
-    const attachments = context.activity.attachments?.filter((a) => !a.contentType.startsWith('text/html'))
-    if (!attachments || attachments.length === 0) {
-      return Promise.resolve([])
-    }
-
-    const connectorClient : ConnectorClient = context.turnState.get<ConnectorClient>('connectorClient')
-    this._httpClient.defaults.headers = connectorClient.axiosInstance.defaults.headers
-
-    const files: InputFile[] = []
-    for (const attachment of attachments) {
-      const file = await this.downloadFile(attachment)
-      if (file) {
-        files.push(file)
-      }
-    }
-
-    return files
-  }
-
-  /**
-     * @private
-     * @param {Attachment} attachment - Attachment to download.
-     * @param {string} accessToken - Access token to use for downloading.
-     * @returns {Promise<InputFile>} - Promise that resolves to the downloaded input file.
-     */
-  private async downloadFile (attachment: Attachment): Promise<InputFile | undefined> {
-    let inputFile: InputFile | undefined
-    if (attachment.contentType === 'application/vnd.microsoft.teams.file.download.info') {
-      const contentSchema = z.object({ downloadUrl: z.string() })
-      const contentValue = contentSchema.parse(attachment.content)
-      const response = await this._httpClient.get(contentValue.downloadUrl, { responseType: 'arraybuffer' })
-      const content = Buffer.from(response.data, 'binary')
-      let contentType = attachment.contentType
-      if (contentType === 'image/*') {
-        contentType = 'image/png'
-      }
-      inputFile = { content, contentType, contentUrl: attachment.contentUrl }
-    } else if (attachment.contentType === 'image/*') {
-      const response = await this._httpClient.get(attachment.contentUrl!, { responseType: 'arraybuffer' })
-      const content = Buffer.from(response.data, 'binary')
-      inputFile = { content, contentType: attachment.contentType, contentUrl: attachment.contentUrl }
-    } else {
-      inputFile = {
-        content: Buffer.from(attachment.content as ArrayBuffer),
-        contentType: attachment.contentType,
-        contentUrl: attachment.contentUrl
-      }
-    }
-    return inputFile
-  }
-
-  /**
-   * Downloads files from the attachments in the current turn context and stores them in state.
-   *
-   * @param context The turn context containing the activity with attachments.
-   * @param state The turn state to store the files in.
-   * @returns A promise that resolves when the downloaded files are stored.
-   */
-  public async downloadAndStoreFiles (context: TurnContext, state: TState): Promise<void> {
-    const files = await this.downloadFiles(context)
-    state.setValue(this._stateKey, files)
-  }
+export class TeamsAttachmentDownloader extends AppTeamsAttachmentDownloader {
```

Comment:
instead of empty class, why not just re-export the one from agents-hosting?

If this is because we want the deprecation warning to appear, that's fine.